interface/forms/vitals/templates/vitals/general_new.php

```diff
@@ -7,7 +7,7 @@
     <?php 
       html_header_show();       
       // Include Bootstrap and datetimepicker
-      call_required_libraries($bootstrap=true,$fancybox=false,$knockout=false,$datepicker=true);
+  call_required_libraries("jquery-min-3-1-1","bootstrap","datepicker");
```

Comment:
I don't see a jquery in the older version. This must be here for the new bootstrap changes.